packages/notebook/src/panel.ts

```diff
@@ -132,6 +136,21 @@ class NotebookPanel extends Widget {
   }
 
   /**
+   * A promise that resolves when the notebook panel is ready.
+   */
+  get ready(): Promise<void> {
+    return this._context.ready.then(() => {
+      let promises: Promise<void>[] = [];
+      each(this.notebook.widgets, widget => {
+        if (widget instanceof MarkdownCell) {
+          promises.push(widget.ready);
```

Comment:
Could other cell types eventually have ready promises? Maybe wait for them all and for now have the others resolve immediately?